tests/fast-integration/skr-test/commerce-mock/index.js

```diff
@@ -22,7 +22,7 @@ const {
 
 const AWS_PLAN_ID = '361c511f-f939-4621-b228-d0fb79a1fe15';
 
-const auditLogsThreshold = 4;
+const auditLogsThreshold = 6;
```

Comment:
could you please document in the code with a commentary why the threshold has exactly this value?

Reply:
I added a comment. Not super happy with it but it's difficult to formulate because we just checked what the current result of the prometheus query is and then chosen the next higher number.